rust/src/delta_arrow.rs

```diff
@@ -39,6 +40,7 @@ impl From<&schema::SchemaDataType> for ArrowDataType {
     fn from(t: &schema::SchemaDataType) -> Self {
         match t {
             schema::SchemaDataType::primitive(p) => {
+                let decimal_regex = Regex::new(r"\((\d{1,2}),(\d{1,2})\)").unwrap();
```

Comment:
we should wrap this with lazy static to repetitive compilation: https://docs.rs/regex/1.4.3/regex/#example-avoid-compiling-the-same-regex-in-a-loop